server/services/caldav/lib/calendar/calendar.syncUser.js

```diff
@@ -0,0 +1,54 @@
+const logger = require('../../../../utils/logger');
+
+/**
+ * @description Start user sync.
+ * @param {Object} userId - Gladys user to connect & sync.
+ * @returns {Promise} Resolving.
+ * @example
+ * syncUser(user.id)
+ */
+async function syncUser(userId) {
+  const account = await this.connect(userId);
+
+  // Clean all dav Calendar for full Sync
+  let gladysCalendars = await this.gladys.calendar.get(userId);
+  let davCalendars = gladysCalendars.filter((calendar) => calendar.service_id === this.serviceId);
+
+  await Promise.all(
```

Comment:
Same here, be careful with concurrency

Reply:
I understand on the previous case but here we already have all data, there is no request all is executed locally, is it necessary to limit concurrency ?